test/module/index.js

```diff
@@ -98,44 +98,40 @@ describe('Module Tests', function() {
   it('should throw an error when the first parameter is not defined', function() {
     const lighthouse = require('../..');
     return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is not a string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the first parameter is an empty string', function() {
     const lighthouse = require('../..');
-    return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+    return lighthouse('')
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is an empty string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the first parameter is not a string', function() {
     const lighthouse = require('../..');
     return lighthouse({})
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is not a string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the second parameter is not an object', function() {
     const lighthouse = require('../..');
-    return lighthouse(VALID_TEST_URL, [])
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+    return lighthouse(VALID_TEST_URL, 'flags')
```

Comment:
yeah, I'm not sure what the best thing is here. An array technically operates fine when passed in, so I guess that's fine. I just picked something that would fail. Happy to change